packages/siri-sx-stats-generator/utils/statGenerators.ts

```diff
@@ -46,6 +51,18 @@ export const generateConsequenceStats = (key: string, disruption: Disruption) =>
                         : acc[key].stopsConsequencesCount,
                 stopsAffected:
                     consequence.consequenceType === "stops" ? consequence.stops.length : acc[key].stopsAffected,
+                ...(cancelFeatureFlag
+                    ? {
+                          journeysConsequencesCount:
+                              consequence.consequenceType === "journeys"
+                                  ? acc[key].journeysConsequencesCount + 1
+                                  : acc[key].journeysConsequencesCount,
+                          journeysAffected:
+                              consequence.consequenceType === "journeys"
+                                  ? consequence.journeys.length
+                                  : acc[key].journeysAffected,
```

Comment:
```suggestion
                          journeysAffected:
                              consequence.consequenceType === "journeys"
                                  ? acc[key].journeysAffected + consequence.journeys.length
                                  : acc[key].journeysAffected,
```

I think there's a bug with the stats generator and it's not been calculating stopsAffected and servicesAffected correctly. The calculation to get journeys affected should be as above, we should modify the services and stops calculation to follow this logic too.